recipes/fast-dds/all/conanfile.py

```diff
@@ -153,20 +155,21 @@ def package_info(self):
 
         # component fastrtps
         self.cpp_info.components["fastrtps"].set_property("cmake_target_name", "fastrtps")
-        self.cpp_info.components["fastrtps"].libs = tools.collect_libs(self)
+        self.cpp_info.components["fastrtps"].set_property("cmake_target_aliases", ["fastdds::fastrtps"])
```

Comment:
what is the reason for adding an alias?

Reply:
It seems that it doesn't occur anywhere except in conan-center-index where 'fastdds::fastrtps' is found. However, it was aliased in the previous recipes, so I kept it.